local_groups/models.py

```diff
@@ -33,6 +33,21 @@ class Group(models.Model):
         blank=False,
         unique=True
     )
+
+    GROUP_TYPES = (
+        ('state-organizing-committee', 'State Organizing Committee'),
+        ('state-chapter', 'State Chapter'),
+        ('campus', 'Campus'),
+        ('local-group', 'Local Group')
```

Comment:
I generally suggest to use integer IDs for stuff like this. The point of the key is that it should be a unique ID but doesn't need to be human readable. Plus then if we change the display name we don't have to worry about keeping the key in sync.

Reply:
Agreed, except we need these values in the exported CSV so used strings here, otherwise we get group type 1, 2, 3, or 4 without changing export code.

Reply:
"we need these values in the exported CSV" - what do you mean by this? why do we need it?

Reply:
Organizing team exports CSVs of groups for various organizing purposes.

Reply:
Ok sure. I suggest adding code comment here to explain this, it's not really intuitive that these need to be semantic human readable keys for this reason. Also we could potentially use integer IDs and document what the id numbers correspond to. Or maybe we can insert the display name into csv, or potentially both the key and the display name.